library/ssl_tls13_client.c

```diff
@@ -1641,6 +1642,12 @@ static int ssl_tls13_postprocess_server_hello( mbedtls_ssl_context *ssl )
             goto cleanup;
     }
 
+    MBEDTLS_SSL_DEBUG_MSG( 3,
+            ( "Server selected key exchange mode: %s",
+              handshake->key_exchange_mode == 1 ? "psk" :
+              (handshake->key_exchange_mode == 2 ? "ephemeral" :
+              "psk_ephemeral")) );
+
```

Comment:
And insert the string to line 1625, 1630, 1635.
we SHOULD not use hard code integer const.
```suggestion
```